src/deploy/functions/release/fabricator.ts

```diff
@@ -545,9 +560,12 @@ export class Fabricator {
     endpoint: backend.Endpoint & backend.BlockingTriggered,
     update: boolean
   ): Promise<void> {
-    await this.executor
-      .run(() => authBlocking.registerTrigger(endpoint, update))
-      .catch(rethrowAs(endpoint, "register blocking trigger"));
+    this.triggerQueue = this.triggerQueue.then(async () => {
+      await this.executor
+        .run(() => services.serviceForEndpoint(endpoint).registerTrigger(endpoint, update))
+        .catch(rethrowAs(endpoint, "register blocking trigger"));
+    });
```

Comment:
Why don't we make the `triggerQueue` be a feature of the Auth service? That way we run auth trigger updates in serial but can run other trigger updates in parallel.

Reply:
I was trying to avoid defining a class for the Auth Blocking service, but parallelizing makes a lot of sense here, I'll make the change